src/app/containers/Ad/index.jsx

```diff
@@ -12,8 +12,9 @@ const AdContainer = ({ slotType }) => {
   const { ads } = useContext(ServiceContext);
   const hasAds = pathOr(false, ['hasAds'], ads);
   const { enabled: adsEnabled } = useToggle('ads');
+  const { enabled: ampAdsEnabled } = useToggle('ampAds');
 
-  if (!hasAds || !adsEnabled) {
+  if (!hasAds || !adsEnabled || (isAmp && !ampAdsEnabled)) {
```

Comment:
This isn't quite the logic we wan't - if `!adsEnabled`, then we don't want to render anything. If adsEnabled is true, then we need to check if amp ads are enabled. Maybe we should just check the ampAdsEnabled toggle in the amp ad container?

Reply:
Argh! Yeah, didn't see that!